opendebates/forms.py

```diff
@@ -83,3 +86,81 @@ def save(self, commit=True):
 class OpenDebatesAuthenticationForm(AuthenticationForm):
     username = forms.CharField(max_length=254,
                                label="Username or Email")
+
+
+class MergeFlagForm(forms.ModelForm):
+    duplicate_of_url = forms.URLField()
+
+    class Meta:
+        model = Flag
+        fields = ('duplicate_of_url', )
+
+    def __init__(self, *args, **kwargs):
+        self.idea = kwargs.pop('idea')
+        self.voter = kwargs.pop('voter')
+        super(MergeFlagForm, self).__init__(*args, **kwargs)
+
+    def clean_duplicate_of_url(self):
+        # parse the URL and use Django's resolver to find the urlconf entry
+        path = urlparse(self.cleaned_data['duplicate_of_url']).path
+        try:
+            url_match = resolve(path)
+        except Resolver404:
+            url_match = None
+
+        if not url_match or url_match.url_name != 'vote':
```

Comment:
Look at the testing front page: https://testing.demopenforum.com/

If you click on any of the links in the 'Recent Activity' sidebar, it takes you to the 'vote' link, so that's the one I used. I need to fix this to accept either one.